tests/test_jx/test_metadata.py

```diff
@@ -111,7 +111,7 @@ def test_get_nested_columns(self):
                         {"b": "x", "v": 2},
                         {"b": "y", "v": 3}
                     ]},
-                    {"o": 2, "_a": {"b": "x", "v": 5}},
+                    {"o": 2, "_a": [{"b": "x", "v": 5}]},
```

Comment:
Since jx expressions make no distinction between a list with one element, and the element on its own, the latter must always be chosen.  Use `unwraplist` on these.